src/plugins/intel_cpu/src/nodes/executors/acl/acl_conv.cpp

```diff
@@ -98,6 +98,11 @@ bool ACLConvolutionExecutor::supports(const ConvConfig& config) {
         VERIFY(config.descs.at(ARG_BIAS)->getPrecision() == ov::element::i32, UNSUPPORTED_BIAS_PRECISIONS);
     }
     VERIFY(config.attrs.postOps.size() <= 1U, UNSUPPORTED_BY_EXECUTOR);
+    // if output precision is quantized and FQ is not fused (i.e. requantize scale can not be applied)
+    // then the executor is not selected because of accuracy degradation (int32 accumulator value is quantized by trivial scale)
+    if (config.descs.at(ARG_DST)->getPrecision().is_quantized()) {
```

Comment:
Can we combine this check with `isQuantized`? From my perspective, it is clearer: the `isQuantized` will define the following requirements for `ACLConvolutionExecutor`:
1. low precision inputs
2. low precision output with requantization fused (=FakeQuantizePostOp exists)